compiler/rustc_ast_pretty/src/pprust/state/expr.rs

```diff
@@ -528,6 +530,18 @@ impl<'a> State<'a> {
                 self.word("asm!");
                 self.print_inline_asm(a);
             }
+            ast::ExprKind::FormatArgs(fmt) => {
+                self.word("format_args!");
```

Comment:
This is not correct, as discussed in the zulip thread, neither `asm!` nor `format_args!` should do this, but should use some canonical expanded token-based representation instead.
If you don't plan to implement it in this PR (I admit it's not super urgent), then it needs at leas a FIXME and comment.

Reply:
Ah, right. Expanding `std::arch::asm!()` now results in just `asm!()`, which is a bit odd. Maybe it should be something like `builtin#asm()` and `builtin#format_args()`, but that probably needs some bikeshedding.